azure-java/Pulumi.yaml

```
@@ -2,4 +2,9 @@ name: ${PROJECT}
 description: ${DESCRIPTION}
 runtime: java
 template:
-  description: A minimal Azure Native Java Pulumi program
+  description: "A minimal Azure Native Java Pulumi program"
```

Nit: I think the quotes are not idiomatic, especially if we have `config.azure-native:location.description` unquoted.
```suggestion
  description: A minimal Azure Native Java Pulumi program
```